File: features/payment/payment/src/main/java/com/tokopedia/payment/router/IPaymentModuleRouter.java
package com.tokopedia.payment.router;

import android.content.Intent;

import java.util.Map;

/**
 * @author anggaprasetiyo on 7/11/17.
 */

public interface IPaymentModuleRouter {

    String getBaseUrlDomainPayment();

    String getGeneratedOverrideRedirectUrlPayment(String originUrl);

    Map<String, String> getGeneratedOverrideRedirectHeaderUrlPayment(String originUrl);

    boolean getEnableFingerprintPayment();
}

File: features/payment/payment/src/main/java/com/tokopedia/payment/router/IPaymentModuleRouter.kt
package com.tokopedia.payment.router

/**
 * @author anggaprasetiyo on 7/11/17.
 */
interface IPaymentModuleRouter {
    val baseUrlDomainPayment: String?
    fun getGeneratedOverrideRedirectUrlPayment(originUrl: String): String?
    fun getGeneratedOverrideRedirectHeaderUrlPayment(originUrl: String): Map<String, String>
    val enableFingerprintPayment: Boolean
}